### PR TITLE
ZK-4783: Modify error message box CSS design (provide scrollbar while word-break fail)

### PR DESCRIPTION
**Pull request by b952426**
ZK-4783: Modify error message box CSS design (provide scrollbar while word-break fail)

### CODE REVIEW

File: zktest/test/java/org/zkoss/zktest/zats/test2/F96_ZK_4783Test.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.zkoss.zktest.zats.WebDriverTestCase;
+import org.zkoss.zktest.zats.ztl.JQuery;
+
+public class F96_ZK_4783Test extends WebDriverTestCase {
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/F96_ZK_4783Test.java:19]: 1. extract test method
2. comma space

File: zktest/src/archive/test2/F96-ZK-4783.zul [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	Description:
+
+	History:
+		Fri Mar 12 10:03:11 CST 2021, Created by katherinelin
+
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+-->
+<zk>
+	<hlayout>
+		<a id="breeze" href="?zktheme=breeze" label="Breeze" />
```
Comment (scribetw) [inline zktest/src/archive/test2/F96-ZK-4783.zul:16]: use template

**Review by plijyfes — APPROVED** [review]

**Review by DevChu — APPROVED** [review]

File: zktest/src/archive/test2/F96-ZK-4783.zul [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		Fri Mar 12 10:03:11 CST 2021, Created by katherinelin
+
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+-->
+<zk>
+	<apply templateURI="/test2/include/themes.zul"/>
+	<label multiline="true">
+		1. Test with all themes.
+		2. While word-break fail(alert 1 and messagebox 1), should provide scrollbar.
+		3. messagebox 3 width shoud be 600px.
```
Comment (scribetw) [inline zktest/src/archive/test2/F96-ZK-4783.zul:19]: ?

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**